### PR TITLE
Added Karger's Algorithm (Min Cut) in C++ and Python.

### PR DESCRIPTION
**Pull request by thakurdiwakar**
## 🛠️ Issue (Number)

Issue no #767 

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
Screenshot (98)
Screenshot (99)

### CODE REVIEW

File: C++/Karger's Algorithm (Min Cut)/kargers.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#include <iostream>
```
Comment (Kumar-laxmi) [inline C++/Karger's Algorithm (Min Cut)/kargers.cpp:1]: Please don't create separate folder for the algorithm, instead just include the Program file under language's Graph folder with README.md content commented in program file

File: Python/Karger's Algorithm (Min Cut)/kargers.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import random
```
Comment (Kumar-laxmi) [inline Python/Karger's Algorithm (Min Cut)/kargers.py:1]: Please don't create separate folder for the algorithm, instead just include the Program file under language's Graph folder with README.md content commented in program file

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

File: C++/Karger's Algorithm (Min Cut)/kargers.cpp (hunk 1 above)
Comment (thakurdiwakar) [reply C++/Karger's Algorithm (Min Cut)/kargers.cpp:1]: Okay

File: Python/Karger's Algorithm (Min Cut)/kargers.py (hunk 2 above)
Comment (thakurdiwakar) [reply Python/Karger's Algorithm (Min Cut)/kargers.py:1]: Okay

**Comment by thakurdiwakar** [comment]
I have include the Program file under language's Graph folder

**Review by zugzwang03 — APPROVED** [review]

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Please provide the screenshot of output and not the code

**Comment by thakurdiwakar** [comment]
> Please provide the screenshot of output and not the code

Added screenshot ,check @Kumar-laxmi

**Review by zugzwang03 — COMMENTED** [review]
remove bloom_filter files

**Comment by thakurdiwakar** [comment]
> remove bloom_filter files
Done

**Comment by thakurdiwakar** [comment]
Please review @Kumar-laxmi

**Review by 072arushi — APPROVED** [review]

**Review by Arun9739 — APPROVED** [review]

**Pull request closed without merging**